### ISSUE TITLE
Crash on first try!

### ISSUE THREAD

**Issue by npandita19** [sixty-north/cosmic-ray#205]
###  Cosmic-ray crashed on first attempt itself, so not sure what you are looking for. These are the steps I followed.
$cosmic-ray init --timeout=10 AST_test_session astmonkey astmonkey/tests/
$ cosmic-ray exec AST_test_session
failed to create process.
Traceback (most recent call last):
  File "C:\Python34\Scripts\cosmic-ray-script.py", line 11, in <module>
    load_entry_point('cosmic-ray==1.1.0a1', 'console_scripts', 'cosmic-ray')()
  File "C:\Python34\lib\site-packages\cosmic_ray-1.1.0a1-py3.6.egg\cosmic_ray\cli.py", line 380, in main
    sys.exit(handler(sub_config))
  File "C:\Python34\lib\site-packages\cosmic_ray-1.1.0a1-py3.6.egg\cosmic_ray\cli.py", line 186, in handle_exec
    cosmic_ray.commands.execute(db, dist)
  File "C:\Python34\lib\site-packages\cosmic_ray-1.1.0a1-py3.6.egg\cosmic_ray\commands\execute.py", line 58, in execute
    for result in results:
  File "C:\Python34\lib\site-packages\cosmic_ray-1.1.0a1-py3.6.egg\cosmic_ray\commands\execute.py", line 16, in local_executor
    timeout)
  File "C:\Python34\lib\site-packages\celery-3.1.25-py3.6.egg\celery\local.py", line 188, in __call__
    return self._get_current_object()(*a, **kw)
  File "C:\Python34\lib\site-packages\celery-3.1.25-py3.6.egg\celery\app\task.py", line 428, in __call__
    return self.run(*args, **kwargs)
  File "C:\Python34\lib\site-packages\cosmic_ray-1.1.0a1-py3.6.egg\cosmic_ray\tasks\worker.py", line 49, in worker_task
    result = json.loads(outs)
  File "C:\Python34\lib\json\__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "C:\Python34\lib\json\decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Python34\lib\json\decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

Any leads on how can I bypass this would be helpful.

**Comment by abingham** [comment]
I think the first thing to do is to see what the worker subprocess is returning. In `cosmic_ray.tasks.worker.worker_task` try using this code:
```
proc = subprocess.Popen(command,
                            stdout=subprocess.PIPE,
                            stderr=subprocess.PIPE,
                            universal_newlines=True)
try:
    outs, errs = proc.communicate(input=None, timeout=timeout)
    print('stdout:', outs)
    print('stderr:', errs)
    result = json.loads(outs)
except subprocess.TimeoutExpired as e:
    result = ('timeout', e.timeout)
        proc.kill()
``` 
in place of the the POpen stuff that's already there. This should print out what the worker process is returning.

**Comment by npandita19** [comment]
@abingham Thanks! the logs pointed out that pydot was missing, I installed pydot and CR has started mutation testing. Will update and hopefully close the issue once the execution completes. Any estimate on how long it may take? I am testing it on a codebase with roughly 1000 LoC

**Comment by abingham** [comment]
When i ran the astmonkey tests under CR it took about 20 minutes. However,
I noticed that the tests didn't seem to pass even without mutation. If
that's true then you won't get meaningful results from CR.

**Comment by npandita19** [comment]
@abingham That's the first thing I checked. I ran nose2 on ASTMonkey code and all tests passed. 

$My_PATH\astmonkey-master>nose2
......................s..s.......................s...s......s.........s
----------------------------------------------------------------------
Ran 71 tests in 0.084s

OK (skipped=6)

And these are the results from CR too.

Report:-
total jobs: 823
complete: 823 (100.00%)
**survival rate: 0.00%**

Thanks a lot for your help and great job with CR
